### PR TITLE
 The Next Web

### PR DESCRIPTION
**Pull request by BrianSammit**
The project consists of creating a responsive website of The Next Web that changes the position and size of the items and images.

we use HTML and CSS to go through the project using "media query" to resize the items on the page in the mobile screen, tablet screen, and desktop screen.

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,1148 @@
...
+  height: 60px;
+}
+
+.pay {
+  position: absolute;
+  top: 10px;
+  left: 10px;
+  width: 60px;
+  height: 60px;
+}
```
Comment (MuriloRoque) [inline css/styles.css:538]: - [ ] Please avoid repeating the same properties, put them all together :wink:

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,1148 @@
...
+    margin: 0 5px;
+  }
+
+  .fork div {
+    width: 354px;
+  }
+
+  .article-latest {
+    width: 354px;
+  }
```
Comment (MuriloRoque) [inline css/styles.css:942]: - [ ] Same problem as above, repeating properties.

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
# Status: Changes requested :red_circle: 

## Hello @BrianSammit and @alvinlouis29 :wave: 

## You have done an amazing job so far :grin: , it's hard to spot differences between your page and the original. However, I still noticed some details that I need you to fix :eyeglasses:  Please, let me help you:

# Page

- I really liked the effects when changing screen sizes :smile: 

- [ ] You need to add the TNW logo inside of the big image to the left, just like the original. See below:

## Yours:
Screenshot from 2020-08-14 15-03-15

## Original:
Screenshot from 2020-08-14 15-03-22
<br>

## NOTE: This logo is ONLY present in large screens :warning: 

<br> 

- [ ] In the Latest News section, you need to fill the entire width of the screen with the articles. In your transition from medium to a large screen, they are not centered as the original because of this:
## Yours:
Screenshot from 2020-08-14 15-16-37
Screenshot from 2020-08-14 15-17-09

## Orignal:

Screenshot from 2020-08-14 15-16-46
Screenshot from 2020-08-14 15-17-12
<br>
- [ ] The same thing happens in the Latest Funding section, they are not aligned in the transition between medium and large screens:
## Yours:

Screenshot from 2020-08-14 15-27-59
## Original:
Screenshot from 2020-08-14 15-27-56
<br>

- [ ] I will not repeat myself in this review. But the same issue above is happening in the next two sections, please fix them! :wink: 

## General

- Nice job with the README file, looks very professional :+1: 
- Good PR and commit messages :heavy_check_mark: 

## Section

- [ ] Please, check your live link in the README file, I believe it's not the same as the one I cloned. I saw some differences.
- Your HTML is awesome, keep up the good work!

- [ ] I made some comments in your CSS file, please take a look :grin: 

# Summary

- You are in the right direction, everything looks in place, just some minor fixes and you will be good to go :rocket: 

## Kindly make the requested changes and submit for another review, please :pray: 

## I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Comment by BrianSammit** [comment]
We fix the sizing problems on the page and we add the image to the left on large screens.
Also, we take out the repeating properties.

**Review by MuriloRoque — APPROVED** [review]
# Status: Approved :green_circle: 

## Hello again @BrianSammit and @alvinlouis29 :wave: 

## Thank you for making the required changes I asked you. Congratulations, your project is approved :clap: You may merge! 

## Note: Please update your live link :grin: :warning: 

## Good luck in your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**